include/caf/click_detector.h

```diff
@@ -7,6 +7,9 @@
 #ifndef _CLICK_DETECTOR_H_
 #define _CLICK_DETECTOR_H_
 
+#include <stdint.h>
```

Comment:
Thanks for the contribution

Please align the commit message to our compliance requirements (https://github.com/nrfconnect/sdk-nrf/actions/runs/12162571516/job/34019513199?pr=19262):

- Add `Signed-off-by` line
- Prefix commit title: `caf: click_detector: Add missing includes to click_detector.h`
- Add commit body

As soon as our CI tests pass, we will be able to merge the change